frontend/src/Pages/TakeAction/ScrollSpy/ScrollSpy.tsx

```diff
@@ -1,31 +1,79 @@
-import { Container } from "react-bootstrap";
-import DesktopProgressBar from "./DesktopScrollSpy/DesktopScrollSpy";
-import MobileProgressBar from "./MobileScrollSpy/MobileScrollSpy";
-import styles from "./ScrollSpy.module.css";
+import cx from "classnames";
+import { useEffect, useState } from "react";
+import { Card, ProgressBar } from "react-bootstrap";
+import styles from "./MobileScrollSpy.module.css";
+import Step from "./Step";
 
 type Props = {
     isEmailSent: boolean;
     isPhoneCallMade: boolean;
     isSocialPosted: boolean;
+    desktop?: boolean;
 };
 
-export default function ScrollSpy({ isEmailSent, isPhoneCallMade, isSocialPosted }: Props) {
+export default function MobileScrollSpy({ isEmailSent, isPhoneCallMade, isSocialPosted, desktop = false }: Props) {
```

Comment:
I left a high-level comment that might make this obsolete, but this is being used for both desktop and mobile. Maybe just rename this to `ScrollSpy`? (That's how it's used outside of this file, too.)

```suggestion
export default function ScrollSpy({ isEmailSent, isPhoneCallMade, isSocialPosted, desktop = false }: Props) {
```